### DISCUSSION TITLE
Not crawling daily.

### DISCUSSION THREAD

**Discussion by FrigginYeah** [towfiqi/serpbear#82] [General]
I've deployed on docker. When initially deployed it would crawl data. Now I just checked and it had not crawled for weeks but did crawl when I initiated a keyword refresh. So the crawling seems to work but not sure where to get started troubleshooting crawling frequency as I do not see that exposed anywhere. 

Any help?

**Comment by towfiqi** [comment]
Sorry for the late reply. As you may have noticed, this option has already been added in version 0.2.5
